doc/api_assets/style.css

```diff
@@ -598,7 +598,8 @@ hr {
   padding-left: 5rem;
 }
 
-#toc .stability_0::after {
+#toc .stability_0::after,
+.deprecated-inline::after {
```

Comment:
Holding off on approving until I have a chance to look at the impact of this change. If it affects other things than what's in this PR, a before/after screenshot might be helpful.

Reply:
The added CSS selector only applies to the new `<span class="deprecated-inline"></span>`.

<details>
<summary>Light mode</summary>

![screenshot of the new `deprecated-inline` CSS class](https://user-images.githubusercontent.com/3109072/189545628-4cbf082b-9376-458e-9320-9d0b349632ef.png)

</details>

<details>
<summary>Dark mode</summary>

![screenshot of the new `deprecated-inline` CSS class in dark mode](https://user-images.githubusercontent.com/3109072/189545398-b5f6f5ce-980a-49dd-81ae-cb7cca06e3f3.png)

</details>

(The same rule already applies to deprecated items in the TOC.)